source/server/services/user-creation-service.js

```diff
@@ -9,16 +9,35 @@ Space.Object.extend(Space.accounts, 'UserCreationService', {
     Space.messaging.EventPublishing
   ],
 
-  statics: {
-    handleUserCreation(options, user) {
-      // Assign guids as user ids
-      user._id = options.userId;
+  onDependenciesReady() {
```

Comment:
There was a good reason why i put the `onCreateUser` callback into `onExtending` :wink:
We only want to register the callback once with Meteor – no matter how many instances of the `UserCreationService` are created, e.g during tests!

`onExtending` is called once when the class was setup / created. That's also the reason why i used a static handler.